src/http.rs

```diff
@@ -51,7 +53,7 @@ impl HttpClient {
             .send()?
             .text()?;
 
-        TxLogResponse::deserialize(resp)
+        Ok(Deserialize::deserialize(&Edn::from_str(&resp)?)?)
```

Comment:
`map_err(|e| e.into()) then looks better`

Reply:
Okay then... I prefer the `Ok(...?)` because the conversion is automatic, but I can change that, no problem

Reply:
I don't like the syntax `Ok(...?)` because it looks like bad type conversions. However, we could implement from for the Result of `Edn::from_str(&resp)` to `Result<Self, CruxError>`?

Reply:
Wow I didn't thought about this! Well, for now I will follow with the `map_err(|e| e.into())` code style, in another PR we can try this okay?